### PR TITLE
Tidy up `NEWS` file

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Via https://github.com/lorenzwalthert/stylermd

New version of the `NEWS` file: https://github.com/r-lib/lintr/blob/tidy_news_file/NEWS.md

### CODE REVIEW

**Comment by MichaelChirico** [comment]
could you summarize what changes are being made here?

I don't really see a need to do this (one more purely stylistic thing to maintain), but maybe there's a smaller part that's useful?

cc @AshesITR

**Comment by jimhester** [comment]
I don't think this is a good idea, as it destroys / obscures the history of the news file. In general I prefer the style of maintaining a sentence per line and using soft wrapping if needed in your editor for markdown prose.
